testing/scripts/testing-script-template.md

```diff
@@ -0,0 +1,30 @@
+# Testing Script Template
+
+## Template
+
+### [#.#.# - WCAG Title](https://www.w3.org/WAI/WCAG21/quickref/#)
+
+What accessibility experience does this reference? This doesn't have to be WCAG. Update link to include reference.
+
+#### Proposed success criteria
+
+What needs to happen for it to be accessible? A 1–2 sentence statement describing the successful behavior.
+
+Where should this be tested? The second paragraph describes on what level of the test is to be applied. For example, is this a behavior we expect to happen on the whole application, in some part of the application, in multiple individual parts of the application, or the like.
+
+#### Proposed testing script
+
+| Step | Expected Behavior | 
+|-----|-----|
+| 1. What needs to happen to get from the beginning to the end? Steps are numbered | What happens if things go correctly? Expected behavior describes successful checkpoints for this step in the script.|
+| 2.  | |
+| 3.  | | 
+| 4.  | |
+| 5.  | |
+| 6.  | |
+| 7.  | |
+| 8.  | |
+| 9.  | | 
+| 10. | | 
+
+Scripts can have as many or as few steps as needed to complete the task. Some steps may not need expected behavior, but try to add expected behavior as consistently as possible.
```

Comment:
```suggestion
Scripts can have as many or as few steps as needed to complete the task. Some steps may not need expected behavior, but try to add expected behavior as consistently as possible.

```

Reply:
Is this missing a comment, or am I missing the suggestion? I don't see what I'm supposed to fix.

Reply:
it is marked as outdated so I assume the suggestion is not applicable all good now